src/rich-text-editor.component.js

```diff
@@ -205,6 +205,16 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     divRef.current.focus()
     setFocused(true)
   }
+
+  function getBrowserDefaultStyling() {
+    if (props.placeholderColor) {
+      return `color: ${props.placeholderColor};`
+    } else if (!!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime)) {
+      return `color: rgb(117, 117, 117);`
+    } else {
+      return `opacity: 0.54;`
+    }
+  }
```

Comment:
<img width="1239" alt="image" src="https://user-images.githubusercontent.com/9112811/64309495-da1ec780-cf59-11e9-83b5-dbaf037a459f.png">
Seems to look good with safari (to be honest didn't check IE)

Reply:
the real question is who at mozilla was like "yeah we need it to be .54, not .5"

Reply:
haha